src/pages/ts-language-guide/setup.mdx

```diff
@@ -3,42 +3,5 @@ import { Steps } from "nextra/components"
 
 ## Setup development environment for TypeScript
 
-First install **Node.js** and **npm** to your system by following the official [instructions](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
-
-Typescript programs can be compiled into WASM components using the `componentize-js` and `jco` npm packages. For **Golem** you should use our forked version of these tools, as they pin all the dependencies to the ones required by Golem Cloud, and they also include workarounds for some experimental features.
-
-These tools are using the [StarlingMonkey](https://github.com/bytecodealliance/StarlingMonkey/) JavaScript engine to create WASM components from JavaScript.
-
-The tool packages can be installed globally, but we recommend adding them to your `package.json`.
-
-Projects created with `golem new` will reference the latest versions of the appropriate forked tools.
-
-In case you want setup you project manually, see the steps for using the [Golem TypeScript SDK](/ts-language-guide/golem-ts-sdk).
```

Comment:
This is described in the under the hood section in building, so I think we can make this one as simple as possible